.github/workflows/cpp.yml

```diff
@@ -60,11 +60,15 @@ jobs:
         image:
           - conda-cpp
           - ubuntu-cpp-sanitizer
+          - ubuntu-cpp-skyhook
         include:
           - image: conda-cpp
             title: AMD64 Conda C++
           - image: ubuntu-cpp-sanitizer
             title: AMD64 Ubuntu 20.04 C++ ASAN UBSAN
+          - image: ubuntu-cpp-skyhook
+            ubuntu: 20.04
```

Comment:
@kou I have to this to run the skyhook integrations tests in github workflows along with the other CPP tests.

Reply:
Sorry. I should have said explicitly.
Why do you want to use a separated job for Skyhook instead of adding Skyhook enabled configuration to existing job?

Reply:
Okay, that makes sense. I didn't want to make changes to properly working stuff before. But now I guess I can do it.